src/reducers/list-reducer.ts

```diff
@@ -66,6 +68,25 @@ export function reducer(
             error: failAction.payload.error,
           });
           return;
+        case RESOURCE_DEL_SUCCESS:
+          const successDeleteAction = action as ResourceSuccessAction;
+          const deletedResourceId =
+            successDeleteAction.payload.serviceParameters.id;
+          Object.assign(resourceList, {
+            loading: false,
+            error: null,
+          });
+          if (Array.isArray(resourceList.data)) {
```

Comment:
afaik it will always be an array

Reply:
Just to mention that data is an object or null so maybe we should support array as well of a defined type since we are already matching a type of object with `{ id: '', ...rest}`

Reply:
Resolved and added a new interface